html/home.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="../styles/styles.css">
+        <title>Weather App</title>
+        <script type="text/javascript" src="../javascript/script.js"></script>
+        <link rel="icon" href="favicon.ico">
+    </head>
+
+    <body>
+        <main>
+
+          <!-- a div containing the website title & a location search bar-->
+          <div id="top_div">
+            <p id="title"> Weather.com</p> 
+            <form id="search_form">
+                <input type="text" id="search_field" name="search_field" placeholder="Search location">
+                <input type="submit" value="submit">
+            </form>
+          </div>
+
+          <!-- a div containing several buttons for dif. aspects of weather -->
+          <div id="button_div">
+            <button type="button">Today</button>
+            <button type="button">Hourly</button>
+            <button type="button">10 day</button>
+            <button type="button">Severe Weather</button>
```

Comment:
Good job. For future projects, you might want to look into semantic html tags: https://www.w3schools.com/html/html5_semantic_elements.asp